tests/integration.rs

```diff
@@ -133,6 +136,53 @@ async fn test_incr() {
 }
 
 #[tokio::test]
+#[serial]
+async fn test_incr_by() {
+    type Response = (Value, Value, Value, Value, Value, Value);
+
+    test_compare::<Response>(|p| {
+        p.cmd("SET").arg("incr_by_key_1").arg(2);
+        p.cmd("SET").arg("incr_by_key_2").arg(10);
+        p.cmd("SET").arg("incr_by_key_3").arg("2");
+
+        p.cmd("INCRBY").arg("incr_by_key_1").arg(10);
+        p.cmd("INCRBY").arg("incr_by_key_2").arg("7");
+        p.cmd("INCRBY").arg("incr_by_key_3").arg(-2);
+
+        // Value is not an integer or out of range error.
+        // p.cmd("SET")
+        //     .arg("incr_by_key_4")
+        //     .arg("234293482390480948029348230948");
+        // p.cmd("INCRBY").arg("incr_by_key_4").arg(1);
```

Comment:
@gillchristian I know you don't like commented code but I don't want to deal with these errors right now. We are returning the correct error message but the formats are a bit different. I can uncomment this and make it fail.

Reply:
Yeah, not a fan of commented-out code. I would much rather have the test fail as a TODO that we have to fix.